### PR TITLE
Python: Remove cert_analysis

### PR DESCRIPTION
**Pull request by eranmes**
It's not used and getting in the way of clean-ups.

### CODE REVIEW

**Comment by eranmes** [comment]
Paul, PTAL - it should make cleaning up the Python logobserver code slightly easier.

File: python/ct/proto/certificate.proto [hunk 1]
```diff
@@ -24,6 +24,7 @@ message DNAttribute {
   optional string value = 2;
 }
 
+// TODO(eran): Remove this message.
```
Comment (phad) [inline python/ct/proto/certificate.proto]: You should mark the place where it is used, in X509Description, as deprecated.

File: python/utilities/logobserver/logobserver.py [hunk 2]
```diff
@@ -6,7 +6,6 @@
 import sys
 import requests
 
-from ct.cert_analysis import tld_list
```
Comment (phad) [inline python/utilities/logobserver/logobserver.py]: did you also delete this source file?

File: python/ct/proto/certificate.proto (hunk 1 above)
Comment (eranmes) [inline python/ct/proto/certificate.proto]: Done.

File: python/utilities/logobserver/logobserver.py (hunk 2 above)
Comment (eranmes) [inline python/utilities/logobserver/logobserver.py]: Yes.

**Comment by phad** [comment]
Thanks, LGTM modulo Travis.

**Pull request merged**